LICENSE.md

```diff
@@ -0,0 +1,9 @@
+MIT License
+
+Copyright (c) 2020 Bryan Pan
```

Comment:
Should/can we make this "Creative Labs"? Idk how this legal stuff works but will look it up

Reply:
yeah i didnt know and it was getting late.. not sure how the legality of it works as a student organization..